DEVELOPMENT.rst

```diff
@@ -15,7 +15,7 @@ RDT Technical Overview
 ----------------------
 
 The goal of RDT is to be able to transform data that is not machine learning ready into data that
```

Comment:
Here I would explain what `sdtypes` are. We should probably do this in the contributing guide as well. A brief intro to what it means/represents

Reply:
You can use [this wording](https://datacebo.gitbook.io/rdt/usage/sdtypes) (or a variant of it)